src/Resources/config/services.xml

```diff
@@ -46,5 +50,17 @@
             <tag name="kernel.event_listener" event="sylius.order.pre_create" method="processOrderBeforeCreation" />
             <tag name="kernel.event_listener" event="sylius.order.pre_create" method="completeOrderBeforeCreation" />
         </service>
+
+        <service id="Sylius\AdminOrderCreationPlugin\EventListener\PaymentLinkCreationListener">
+            <argument type="service" id="payum" />
+            <argument type="service" id="sylius.manager.order" />
+            <argument type="service" id="Sylius\AdminOrderCreationPlugin\Sender\OrderPaymentLinkSender" />
+            <argument type="string">sylius_shop_order_after_pay</argument>
+            <tag name="kernel.event_listener" event="sylius.order.post_create" method="setPaymentLink" />
```

Comment:
Does it mean that ```PaymentLinkCreationListener```'s setPaymentLink() method will be called once any order in Sylius is placed?

Reply:
Indeed, it does. I was also thinking about that, that it should be limited only to orders created in admin panel. Will be fixed 👍

Reply:
👍